pkg/controller/controller.go

```diff
@@ -496,7 +497,7 @@ func (p *csiProvisioner) ProvisionExt(options controller.ProvisionOptions) (*v1.
 		return nil, controller.ProvisioningFinished, fmt.Errorf("fstype specified in parameters with both \"fstype\" and \"%s\" keys", prefixedFsTypeKey)
 	}
 	if len(fsType) == 0 {
-		fsType = defaultFSType
+		fsType = p.defaultFSType
```

Comment:
Here's the implementation of `--default-fstype=none`:

```
if fsType == "" && p.defaultFSType != "none" {
    fsType = p.defaultFSType
}
```

Reply:
Done.